### PR TITLE
Enable configfs-tsm interface

### PR DESCRIPTION
**Pull request by jerryrhyu**

### CODE REVIEW

File: go-tdx/gcp_adapter.go [hunk 1]
```diff
@@ -137,6 +127,51 @@ func (adapter *gcpAdapter) CollectEvidence(nonce []byte) (*connector.Evidence, e
...
+	req := &report.Request{
+		InBlob:     reportData[:],
+		GetAuxBlob: false,
+	}
+	resp, err := linuxtsm.GetReport(req)
+	if err != nil {
+		return nil, err
+	}
+	log.Infof("provider is %s", resp.Provider)
+	tdReport := resp.OutBlob
```
Comment (arvind5) [inline go-tdx/gcp_adapter.go:144]: should we rename it to quote?

**Review by arvind5 — APPROVED** [review]

File: go-tdx/gcp_adapter.go [hunk 2]
```diff
@@ -137,6 +127,51 @@ func (adapter *gcpAdapter) CollectEvidence(nonce []byte) (*connector.Evidence, e
...
+
+	req := &report.Request{
+		InBlob:     reportData[:],
+		GetAuxBlob: false,
+	}
+	resp, err := linuxtsm.GetReport(req)
+	if err != nil {
+		return nil, err
+	}
+	log.Infof("provider is %s", resp.Provider)
```
[the remark this answers is not in the document]
Comment (jerryrhyu) [reply go-tdx/gcp_adapter.go]: removed

**Comment by mythi** [comment]
Are you planning to submit this to  `main` also? It's not GCP specific.

**Comment by jerryrhyu** [comment]
> Are you planning to submit this to `main` also? It's not GCP specific.

Yes, we plan to merge to main as well.

**Pull request merged**